packages/components/src/templates/next/layouts/Collection/utils/getCollectionItems.ts

```diff
@@ -11,13 +12,15 @@ export interface GetCollectionItemsProps {
   permalink: string
   sortBy?: CollectionPagePageProps["defaultSortBy"]
   sortDirection?: CollectionPagePageProps["defaultSortDirection"]
+  tagCategories?: CollectionPagePageProps["tagCategories"]
 }
 
 export const getCollectionItems = ({
   site,
   permalink,
   sortBy,
   sortDirection,
+  tagCategories,
 }: GetCollectionItemsProps): AllCardProps[] => {
```

Comment:
nit: can add a TODO comment that we can do the optimisation on getting the filters directly from the UUID rather than doing another pass of the collection items using the `getAvailableFilters`